src/dispatch/static/dispatch/src/search/Table.vue

```diff
@@ -0,0 +1,166 @@
+            :loading="loading"
+            loading-text="Loading... Please wait"
+          >
+            <template #[`item.individuals`]="{ item }">
+              <v-chip small color="info" text-color="white">{{ item.individuals.length }}</v-chip>
+            </template>
+            <template #[`item.teams`]="{ item }">
+              <v-chip small color="info" text-color="white">{{ item.teams.length }}</v-chip>
+            </template>
+            <template #[`item.services`]="{ item }">
+              <v-chip small color="info" text-color="white">{{ item.services.length }}</v-chip>
+            </template>
+            <template #[`item.enabled`]="{ item }">
+              <v-simple-checkbox v-model="item.enabled" disabled />
+            </template>
+            <template #[`item.created_at`]="{ item }">
+              <v-tooltip bottom>
+                <template #activator="{ on, attrs }">
+                  <span v-bind="attrs" v-on="on">{{ item.created_at | formatRelativeDate }}</span>
+                </template>
+                <span>{{ item.created_at | formatDate }}</span>
+              </v-tooltip>
+            </template>
+            <template #[`item.updated_at`]="{ item }">
+              <v-tooltip bottom>
+                <template #activator="{ on, attrs }">
+                  <span v-bind="attrs" v-on="on">{{ item.updated_at | formatRelativeDate }}</span>
+                </template>
+                <span>{{ item.updated_at | formatDate }}</span>
+              </v-tooltip>
+            </template>
+            <template #[`item.data-table-actions`]="{ item }">
+              <v-menu bottom left>
+                <template #activator="{ on }">
+                  <v-btn icon v-on="on">
+                    <v-icon>mdi-dots-vertical</v-icon>
+                  </v-btn>
+                </template>
+                <v-list>
+                  <!-- <v-list-item @click="createEditShow(item)"> -->
+                  <!--   <v-list-item-title>View / Edit</v-list-item-title> -->
+                  <!-- </v-list-item> -->
+                  <v-list-item @click="removeShow(item)">
+                    <v-list-item-title>Delete</v-list-item-title>
+                  </v-list-item>
+                </v-list>
+              </v-menu>
+            </template>
+          </v-data-table>
+        </v-card>
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+
+<script>
+import { mapFields } from "vuex-map-fields"
+import { mapActions } from "vuex"
+
+import DeleteDialog from "@/search/DeleteDialog.vue"
+
+export default {
+  name: "SearchFilterTable",
+
+  components: { DeleteDialog },
+
+  data() {
+    return {
+      headers: [
+        { text: "Name", value: "name", align: "left", width: "10%" },
+        { text: "Description", value: "description", sortable: false },
+        { text: "Individuals", value: "individuals" },
+        { text: "Teams", value: "teams" },
+        { text: "Services", value: "services" },
```

Comment:
Do we need columns with counters for other data types, e.g. Signals?

Reply:
We don't use `SearchFilters` for signals they have their own model `SignalFilters`.

Reply:
Looks like I missed Notification. I'll add it.